trunner/config.py

```diff
@@ -30,13 +30,18 @@ def resolve_phrtos_dir() -> Path:
 PYEXPECT_TIMEOUT = 8
 
 # Available targets for test runner.
-ALL_TARGETS = ['ia32-generic', 'host-pc', 'armv7m7-imxrt106x', 'armv7m7-imxrt117x', 'armv7m4-stm32l4']
+ALL_TARGETS = ['ia32-generic',
+               'armv7a9-zynq7000-emu',
```

Comment:
I'm against adding here targets not defined in `phoenix-rtos-project`. It will:
 - increase confusion regarding the test names
 - break the build functionality of the test runner
 - the third part of the target is for the `project` part of the target

I assume the binaries/images for emulator and target HW might need to be different - but if that would be the case - we need to introduce different targets in `phoenix-rtos-project` repo first.

Reply:
I propose `armv7a9-zynq7000emu` and `armv7a9-zynq7000hw`.

Reply:
nah, the TARGET value should be `${TARGET_FAMILY}-${TARGET_SUBFAMILY}-${PROJECT}` - where `TARGET_SUBFAMILY` usually should mean target CPU. I would probably go for `armv7a9-zynq7000-qemu` and `armv7a9-zynq7000-zedboard` or similar (not sure if we will have multiple qemu targets for `zynq7000` or if `zedboard` is unique enough. Please consult with @HBuczynski regarding how to approach it.

Reply:
Ok, I will consult it and then make changes.

Reply:
I agree with @nalajcie, `armv7a9-zynq7000-qemu` and `armv7a9-zynq7000-zedboard` is fine. There is a one target for `qemu` and the `zedboard` is a unique name for the development board from `digilent`.